### DISCUSSION TITLE
Let's meet up

### DISCUSSION THREAD

**Discussion by Dipeshdahiya** [community/community#142200] [New to GitHub]
### Select Topic Area

General

### Body

Hi myself Dipesh,
I just started my open source journey this month and want to connect to you guys , In general I am very excited to meet up new people and make new friends in this field.
here is my linkedin profile: https://www.linkedin.com/in/dipesh-dahiya/

**Comment by sensie01** [comment]
`Hello, can you mark my answer please`😁

**Comment by Chanduporalla** [comment]
Hi Dipesh, it's great to hear that you've started your open-source journey! Getting involved in open source is a fantastic way to learn, contribute, and connect with others in the tech community. There are many people like you excited to collaborate and share knowledge, so you're definitely in the right place!

Feel free to connect with me or others, and welcome to the open-source world! Best of luck on your journey, and I’m sure you’ll meet many amazing people along the way.

**Comment by Dipeshdahiya** [comment]
Hi @Chanduporalla  nice to meet you

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @Dipeshdahiya 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
